### PR TITLE
Decklist page created

### PR DESCRIPTION
**Pull request by Erakedon**

### CODE REVIEW

File: src/Pages/DeckList/DeckList.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            {id: 8,name: "name3", proto: "bbbbbbbbbb", date: "12 may 2019", author: "niceGuy77"},
+            {id: 13,name: "name3", proto: "bbbbbbbbbb", date: "12 may 2019", author: "niceGuy77"},
+            {id: 9,name: "name4", proto: "cc cccc", date: "6 april 2019", author: "niceGuy77"},
+            {id: 12,name: "name2", proto: "a", date: "6 april 2019", author: "niceGuy77"},
+            {id: 14,name: "name4", proto: "cc cccc", date: "6 april 2019", author: "niceGuy77"},
+        ]
+     }
+
+    inputChanged(event) {
+        console.log(event.target.value);
```
Comment (Erakedon) [inline src/Pages/DeckList/DeckList.js:28]: Console logs will still be usefull while connecting to api

File: src/Pages/DeckList/DeckList.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    }
+
+    redirectToDecklist(id) {
+        console.log(this.props);
+        console.log(id);
+        this.props.history.push("/deck?id=" + id);
+    }
+
+    render() { 
+
```
Comment (NightAngell) [inline src/Pages/DeckList/DeckList.js]: Remove empty line.

File: src/Pages/DeckList/DeckList.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        let resultList = [];
+
```
Comment (NightAngell) [inline src/Pages/DeckList/DeckList.js]: Remove empty line.

File: src/Pages/DeckList/DeckList.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        this.state.searchResult.forEach((deck) => {
+            resultList.push(<DeckResult
+             key={deck.id}  
+             clickHandler={() => {this.redirectToDecklist(deck.id)}}
+             name={deck.name}
+             proto={deck.proto}
+             date={deck.date}
+             author={deck.author}
+             />)
+
```
Comment (NightAngell) [inline src/Pages/DeckList/DeckList.js]: Remove empty line.

**Review by NightAngell — CHANGES_REQUESTED** [review]

File: src/Pages/DeckList/DeckList.js (hunk 1 above)
Comment (NightAngell) [reply src/Pages/DeckList/DeckList.js:28]: Ok

**Review by szefo09 — APPROVED** [review]

**Review by NightAngell — APPROVED** [review]

**Pull request merged**